client/reader/sidebar/test/promo.jsx

```diff
@@ -17,16 +56,17 @@ describe( 'ReaderSidebarPromo', () => {
 
 	test( 'should render the AppPromo when the shouldRenderAppPromo property is true', () => {
 		const adjustedProperties = { shouldRenderAppPromo: true };
-		const wrapper = shallow( <ReaderSidebarPromo { ...adjustedProperties } /> );
-		expect( wrapper.find( '.sidebar__app-promo' ) ).toHaveLength( 1 );
+		const { wrapper } = renderWithRedux( <ReaderSidebarPromo { ...adjustedProperties } /> );
+		expect( wrapper.getElementsByClassName( 'sidebar__app-promo' ) ).toHaveLength( 1 );
```

Comment:
Also you need `container` and not `wrapper`.

Finally, navigating to the root element should be as simple as `container.firstChild`.